.github/workflows/pr-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: PR Lint
+on: pull_request
+
+jobs:
+  markdownlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: install dependencies
+        run: |
+          mkdir markdownlint-cli && cd markdownlint-cli
+          npm install markdownlint-cli
+          cd ../
+      - name: run linter
+        run: |
+          ./markdownlint-cli/node_modules/.bin/markdownlint -i text/0049-continuous-delivery.md -i text/0092-asset-publishing.md text/**/*.md
```

Comment:
Why do we have explicit docs listed here ?

Reply:
The number of changes to conform these files were high. I've taken the time to fix these up now.